tests/unit/coordinator/test_restore.py

```diff
@@ -41,9 +42,11 @@
 )
 
 
+@pytest.mark.parametrize("storage_name", [None, "x", "y"])
```

Comment:
This leads to 3 * 4 = 12 tests. I don't think all of the combinations are really useful?